### PR TITLE
Fix activate.sh 

### LINKED ISSUE #20: GPAW_SETUP_PATH is corrupted when ls is set with coloring enabled
So this one took me a while.

I've got `ls` aliased to `ls --color` in my .bash_profile.

The following line prepends and appends colouring the the path string set at $GPAW_SETUP_PATH. 
https://github.com/conda-forge/gpaw-feedstock/blob/7c3396521450e4b4ffa2d6fda5bb6cf96638dc5c/recipe/activate.sh#L1

This meant that gpaw wasn't finding my files, despite everything seeming fine.

```
(gpaw) [thomasaar@ml7 ~]$ echo $GPAW_SETUP_PATH
/itf-fi-ml/home/thomasaar/.conda/envs/gpaw/share/gpaw

(gpaw) [thomasaar@ml7 ~]$ cd $GPAW_SETUP_PATH
-bash: cd: $'\E[0m\E[38;5;33m/itf-fi-ml/home/thomasaar/.conda/envs/gpaw/share/gpaw\E[0m': No such file or directory
```

Not sure how to suggest a more robust path setting mechanism.

### PR DESCRIPTION
**Pull request by jan-janssen**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

File: recipe/activate.sh [hunk 1]
```diff
@@ -1 +1 @@
-export GPAW_SETUP_PATH=$(ls -rd ${CONDA_PREFIX}/share/gpaw | head -n 1)
+export GPAW_SETUP_PATH="${CONDA_PREFIX}share/gpaw"
```
Comment (marcindulak) [inline recipe/activate.sh]: There is a missing slash between `{CONDA_PREFIX}share`

**Comment by marcindulak** [comment]
Just a comment - would it be possible to add a rudimentary test of gpaw functionality after the build is finished in ci?

```gpaw test```

https://wiki.fysik.dtu.dk/gpaw/install.html#run-a-simple-test-calculation

It would be nice if ci can confirm the basic functionality works.

**Comment by thomasaarholt** [comment]
Brilliant, thanks Jan.

**Comment by jan-janssen** [comment]
@thomasaarholt Sure, I am always happy to help. By the way you can find a list of all the conda package I maintain under https://github.com/jan-janssen/conda-forge-contribution just in case you have some more suggestions.

**Pull request merged**